packages/prettier-plugin-java/src/printers/expressions.js

```diff
@@ -299,16 +298,38 @@ class ExpressionsPrettierVisitor {
       ) {
         suffixes.push(softline);
       }
-      suffixes.push(primarySuffixes[0]);
+      suffixes.push(
+        this.visit(ctx.primarySuffix[0], {
+          shouldDedent:
+            // dedent when simple method invocation
+            countMethodInvocation !== 1 &&
+            // dedent when (chain) method invocation
+            ctx.primaryPrefix[0] &&
+            ctx.primaryPrefix[0].children.fqnOrRefType &&
+            !(
+              ctx.primaryPrefix[0].children.fqnOrRefType[0].children.Dot !==
+              undefined
+            ) &&
+            // indent when lambdaExpression
```

Comment:
indent or dedent ?

Reply:
indent, we should not dedent when we have a lambdaExpression.
The results are not good.